src/main/java/frc/robot/commands/auto/Drive10Feet.java

```diff
@@ -8,24 +8,24 @@
 
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
 import edu.wpi.first.math.trajectory.Trajectory;
 import edu.wpi.first.math.trajectory.TrajectoryConfig;
 import edu.wpi.first.math.trajectory.TrajectoryGenerator;
 import frc.robot.subsystems.Drivetrain;
 
-public class Auto10Feet extends AutoSegmentAbstract {
-  public Auto10Feet(Drivetrain drivetrain, String name) {
-    super(drivetrain, name);
+public class Drive10Feet extends DriveSegmentBaseCommand {
+  public Drive10Feet(Drivetrain drivetrain) {
+    super(drivetrain, 
+        getTrajectory(
+            getDefaultTrajectoryConfig(drivetrain, true)));
```

Comment:
All drive segment commands will initialize similar to this.
It could be cleaner if Java wasn't such a flawed language